README.md

```diff
@@ -65,6 +65,8 @@ print("Average pT for charged pions {0:4.3f}".format(average_pt))
 
 The package will be available as a pre-compiled binary wheels in the future, but for now you have to compile it from source, see next subsection.
 
+On Mac OS X the installation from pre-compiled binary wheels requires command line utils `otool`, `install_name_tool`, and `codesign` from Xcode Command Line Tools to be available on the system. The command `xcode-select --install` installs Xcode Command Line Tools.
+
```

Comment:
I think you cannot do any compilation without XCode Command Line Tools on Mac. Is this note necessary?

Reply:
If `chromo` is installed from wheels  the compilation is not needed. Although it is probably a rare case, it could occur that a user doesn't have Command Line Tools in this case.

Reply:
I don't know the other way how to fix the problem with `Python` library. It seems that the segmention fault occurs because 2 copies are loaded: one from wheel (from machine which built the wheel) and one from the users machine. If only one copy from users machine is used, the problem disappears. So I came up with solution to point to that copy. But for this the fix should be done on user's side.

Reply:
See #126